cli/image.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"fmt"
 
+	gounits "github.com/docker/go-units"
```

Comment:
If we use this package, I think it will be a little bit conflict with a previous commit https://github.com/alibaba/pouch/pull/221 from @Ace-Tang 
Since it does similar thing as go-unit if what I remembered is correct.

Reply:
#221  is a simple function, go-units is powerful package. But，I don't like to use the third party library, so I can replace it to #221 .